### ISSUE TITLE
Small white space at bottom of fullscreen image slider

### ISSUE THREAD

**Issue by kc234** [nicinabox/superslides#323]
There is a small white space at the bottom of my fullscreen image slider despite width and height being set to 100%. Any ideas of how to get rid of it?

**Comment by magic-77** [comment]
Hey,
maybe you forgot to define some basic rules like:

```
body, html {
    height: 100%;
}
body {
    margin: 0px;
}
*, :after, :before {
  box-sizing: border-box;
}
```

**Comment by kc234** [comment]
No. I'm using Bootstrap with container-fluid, so these rules are covered. 

I'm not having this issue anywhere else on the site, just the containers where Superslides is used...?

**Comment by magic-77** [comment]
Hey,
a Link would be helpful. Otherwise it´s difficult to discover the issue
